controls/include/controls/PID.h

```diff
@@ -0,0 +1,38 @@
+//
+// Created by tanvi on 10/30/2020.
+//
+
+#ifndef CONTROLS_PID_H
+#define CONTROLS_PID_H
+
+
+class PID
+{
+private:
+    //double deltaT_;
+    double kP_;
+    double kI_;
+    double kD_;
+    bool angle_;
+
+    double integralError_;
+    double previousError_;
+
+public:
+    PID(double kP, double kI, double kD);
+    PID(double kP, double kI, double kD, bool angle);

```

Comment:
Maybe rename angle to isAngle or something to indicate why, and drop a comment indicating what the argument is for